multiversx_sdk_cli/cli_wallet.py

```diff
@@ -247,3 +270,31 @@ def do_bech32(args: Any):
 
     print(result)
     return result
+
+
+def sign_message(args: Any):
+    message: str = args.message
+    account = cli_shared.prepare_account(args)
+    signable_message = SignableMessage(message, account)
+    signable_message.sign()
+    utils.dump_out_json(signable_message.to_dictionary())
+
+
+def verify_signed_message(args: Any):
+    bech32_address = args.bech32_address
+    verifier = UserVerifier.from_address(Address.from_bech32(bech32_address))
+
+    message: str = args.message
+    verifiable_message = MessageV1.from_string(message)
+
+    signature: str = args.signature
+    if signature.startswith("0x"):
+        signature = signature[2:]
```

Comment:
Indeed, best this way. Possibly handle `0X`, as well (uppercase X)?

Reply:
Done